### DISCUSSION TITLE
Out of VRAM when using GoBig

### DISCUSSION THREAD

**Discussion by Janvonfalken** [hlky/stable-diffusion-webui#836] [General]
I can generate 512x512 images on my 1660 Super with 6GB of VRAM if I use the Turbo optimized mode, but I can't use goBig. Is there a way to make goBig use less VRAM?

**Comment by dudumaroja** [comment]
i was hoping for gobig and golatent cpu support.. i can wait the extra time for sure..

**Reply by TingTingin** [reply]
gobig is just splitting the image up in parts and running it through SD so if you can run SD on cpu then you can use gobig on cpu
